McEngine/src/App/UIFrameworkTest/UIFrameworkTest.cpp

```diff
@@ -0,0 +1,66 @@
+/*
+ * UIFrameworkTest.cpp
+ *
+ *  Created on: May 28, 2017
+ *      Author: Psy
+ */
+
+#include <UIFrameworkTest/UIFrameworkTest.h>
+
+#include "CBaseUIButton.h"
+#include "UIFrameworkTestScreen.h"
+#include "UIFrameworkTestSingleCanvas.h"
+#include "UIFrameworkTestMultiCanvas.h"
+#include "UIFrameworkTestMenu.h"
+
+class UIFrameworkAnchorTestButton : public CBaseUIButton
+{
+public:
+	Vector2 m_vSizeNormal;

```

Comment:
Please always put variables at the bottom of classes. (In this case, below the onMouseOutside() function declaration).